operator/pkg/operator.go

```diff
@@ -155,15 +155,21 @@ func (o *Operator) Start(ctx context.Context) error {
 				o.Logger.Infof("batch %x did not verify. Err: %v", newBatchLog.BatchMerkleRoot, err)
 				continue
 			}
-			responseSignature := o.SignTaskResponse(newBatchLog.BatchMerkleRoot)
+			batchIdentifier := append(newBatchLog.BatchMerkleRoot[:], newBatchLog.SenderAddress[:]...)
+			var batchIdentifierHash = *(*[32]byte)(crypto.Keccak256(batchIdentifier))
+
+			responseSignature := o.SignTaskResponse(batchIdentifierHash)
+			o.Logger.Infof("responseSignature about to send: %x", responseSignature)
```

Comment:
<img width="713" alt="image" src="https://github.com/user-attachments/assets/74361f72-61a8-48ad-a4b2-498b150c30fa">
 
 I don't know if this is very useful, maybe keep it but in debug 
```suggestion
			o.Logger.Debugf("responseSignature about to send: %x", responseSignature)
```